components/brave_wallet_ui/components/desktop/views/market/index.tsx

```diff
@@ -70,47 +59,26 @@ export const MarketView = () => {
   // Custom Hooks
   const swap = useSwap()
   const { swapAssetOptions: tradableAssets } = swap
+  console.log(tradableAssets)
 
-  // Memos
-  const visibleCoinMarkets = React.useMemo(() => {
-    const searchResults = searchTerm === '' ? allCoins : searchCoinMarkets(allCoins, searchTerm)
-    const filteredCoins = filterCoinMarkets(searchResults, tradableAssets, currentFilter)
-    const sorted = sortCoinMarkets(filteredCoins, sortOrder, sortByColumnId)
-    return sorted.slice(0, coinsDisplayCount)
-  }, [allCoins, sortOrder, sortByColumnId, coinsDisplayCount, searchTerm, currentFilter, tradableAssets])
-
-  const onSelectFilter = (value: AssetFilterOption) => {
-    setCurrentFilter(value)
-  }
-
-  const onSort = React.useCallback((columnId: MarketDataTableColumnTypes, newSortOrder: SortOrder) => {
-    const updatedTableHeaders = tableHeaders.map(header => {
-      if (header.id === columnId) {
-        return {
-          ...header,
-          sortOrder: newSortOrder
-        }
-      } else {
-        return {
-          ...header,
-          sortOrder: undefined
-        }
-      }
-    })
-
-    setTableHeaders(updatedTableHeaders)
-    setSortByColumnId(columnId)
-    setSortOrder(newSortOrder)
+  const onSelectCoinMarket = React.useCallback((coinMarket: BraveWallet.CoinMarket) => {
+    dispatch(WalletPageActions.selectCoinMarket(coinMarket))
+    history.push(`${WalletRoutes.Market}/${coinMarket.symbol}`)
   }, [])
 
-  const onShowMoreCoins = React.useCallback(() => {
-    setCoinsDisplayCount(currentCount => currentCount + displayCount)
-  }, [])
+  const onMessageEventListener = React.useCallback((event: MessageEvent<MarketCommandMessage>) => {
+    // validate message origin
+    if (event.origin !== braveMarketUiOrigin) return
 
-  const onSelectCoinMarket = (coinMarket: BraveWallet.CoinMarket) => {
-    dispatch(WalletPageActions.selectCoinMarket(coinMarket))
-    history.push(`${WalletRoutes.Market}/${coinMarket.symbol}`)
-  }
+    const message = event.data
+    switch (message.command) {
```

Comment:
nit: with only one case here we probably don't need this to be a switch.

Reply:
@kdenhartog fixed here: https://github.com/brave/brave-core/pull/12510/commits/26b620f3e2f92724d44f5c1650334b4b197883eb